fbbt/src/ontologies/fbbt-edit.obo

```diff
@@ -63575,6 +63574,260 @@ intersection_of: FBbt:00005106 ! neuron
+is_a: FBbt:00040007 ! synaptic neuropil domain
+relationship: part_of FBbt:00001925 ! embryonic/larval protocerebrum
+property_value: created_by "dos" xsd:string
+creation_date: 2018-02-15T11:10:38Z
+
+[Term]
+id: FBbt:00007722
+name: embryonic/larval nodulus - primordial
+def: "A small primordium that appears as a ventrally directed process of the  lateral  primordial fan-shaped body.  It is demarcated from the surrounding Inferior medial and anterior protocerebrum by different the different concentration of synapses (e.g. as assayed by Bruchpilot expression).  It develops from elements of the DPMm1, DPMpm1/2 and CM4 lineages." [FBC:dw, FlyBase:FBrf0211016]
+is_a: FBbt:00040007 ! synaptic neuropil domain
+property_value: created_by "dos" xsd:string
+creation_date: 2018-02-15T14:21:50Z
+
+[Term]
+id: FBbt:00007723
+name: embryonic/larval posterior lateral protocerebrum
+def: "A synpatic neuropil domain located ventro-postero-laterally in the larval central brain. Anteriorly, the great commissure (BLAv1,BLD5) defines the boundary, in the coronal plane, with the Ventolateral protocerebrum (VLP). Dorsally, there is a slim border with the superior lateral protocerebrum (SLP).   Medially, a virtual sagittal plane intersecting the CP1v tract and the Balp2/3 tract separates the this domain from the ventromedial cerebrum (VMCp).   Laterally, a glial septum and BLP1/2 tract separates it from the lobula." []
+comment: Developmental relation to adult synaptic neuropil domain is a PC from Volker Hartenstein. Note - description of relative location of brain structures in this definition is based on Younossi-Hartenstein et al., 2003 description of the first instar larval brain and Pereanu et al., 2010 third instar larval brain.
+synonym: "centro-posterior medial synaptic neuropil domain" EXACT []
+synonym: "centroposterior medial compartment" EXACT [FlyBase:FBrf0155900]
+synonym: "CPM" EXACT [FlyBase:FBrf0155900]
+synonym: "larval IPp" NARROW [FlyBase:FBrf0211016]
+synonym: "larval posterior inferior protocerebrum" NARROW [FlyBase:FBrf0211016]
+xref: VFB:FBbt_00007070
+is_a: FBbt:00040007 ! synaptic neuropil domain
+relationship: part_of FBbt:00001925 ! embryonic/larval protocerebrum
+property_value: created_by "dos" xsd:string
+creation_date: 2018-02-15T14:33:12Z
+
+[Term]
+id: FBbt:00007724
+name: embryonic/larval posterior ventromedial cerebrum
+def: "The posterior part of the ventromedial cerebrum (BPM).  It borders the posterio inferior protocerebrum (IPp) dorso-medially, the Inferior protocerebrum (IP) dorso-laterally and the posterior lateral protocerebrum (PLP) laterally." []
+synonym: "VMCp" EXACT []
+is_a: FBbt:00040006 ! synaptic neuropil subdomain
+relationship: part_of FBbt:00007063 ! embryonic/larval ventromedial cerebrum
+property_value: created_by "dos" xsd:string
+creation_date: 2018-02-15T14:50:17Z
+
```

Comment:
**embryonic/larval anterior ventromedial cerebrum**
Fixed typos and added in the rest of the info. missed out.
"The anterior part of the ventromedial cerebrum (BPM). It abuts the posterior ventromedial cerebrum (VMCp) at the coronal slice level of the Great Commissure, (GC, BLAv1,BLD5; Pereanu et al. 2010). Ventrally, the boundary with the tritocerebrum (centro-medial and dorsal compartments) is defined by a somewhat curved, more or less axial plane defined by the tracts loVL (Balp2/3; Hartenstein et al., 2015) , loVM (Bamv1/2; Hartenstein et al., 2015) and the BAla3 tract (Kumar et al., 2009). Medially, an extension of the centro-medial tritocerebrum is separated from the VMCa by the loVM (BAmv1/2) fascicle and a glial septum (Pereanu et al., 2006); more posteriorly, this boundary continues as a discontinuity in synaptic density (e.g as assayed using the presynaptic marker bruchpilot). Laterally, the VMCa borders the Lateral Accessory Lobe (LAL), separated by a virtual sagittal plane intersecting the deCP (DALd; posteriorly) and loVM (Bamv1/2; anteriorly) (Pereanu et al., 2006).  At a more posterior level, the Ventro Lateral Protocerebrum (VLP) flanks the VMCa laterally, with the boundary along that of the Posterior Lateral Protocerebrum (PLP)-VMCp, i.e. a virtual sagittal plane intersecting the LEFp (CP1v; posteriorly) and the loVL (Balp2/3; anteriorly) fascicles. The medial Clamp borders the VMCa dorso-laterally with a virtual axial plane intersecting the MEF(CM1,3,4, medially) and LEFp(CP1v, laterally) fascicles constituting the boundary (Hartenstein et al., 2015).  The Crepine borders dorsally, the boundary extending along the DALv2/3 (Hartenstein et al., 2015), DALcl1 and BAmd1v lineage tracts and the commissural fascicle SuEC(DALcl1v; Hartenstein et al., 2015).  The lower toe (medial appendix of larval mushroom body) is dorso-medial. A region of low synaptic density separates the VMCa from the primordial fan-shaped body."